src/components/Page/PagePicker.vue

```diff
@@ -41,6 +41,7 @@
 					class="crumbs-level">
 					<ChevronRightIcon :size="20" />
 					<NcButton type="tertiary"
+						:aria-label="t('collectives', 'Navigate to page {page}', {page: page.title })"
```

Comment:
This seems quite verbose given the label of the surrounding div. In the short run I would suggest to remove the `aria-label` on the surrounding div.
In the long run (?) we might want to use a list with a single label for the breadcrumbs and then one label per button: https://www.w3.org/WAI/ARIA/apg/patterns/breadcrumb/examples/breadcrumb/

I'm also not sure if the `Navigate to page` part is necessary.